Tests/PointFreeTests/__Snapshots__/AtomFeedTests/testAtomFeed.1.Conn.txt

```diff
@@ -2,71 +2,228 @@ GET http://localhost:8080/feed/atom.xml
 Cookie: pf_session={}
 
 200 OK
-Content-Length: 1562
-Content-Type: application/atom+xml
+Content-Length: 7488
+Content-Type: text/xml; charset=utf-8
```

Comment:
Is this a change we want?

Reply:
I guess we've been doing `text/xml` for the public feature huh.

Reply:
yeah seems to be how the RSS feed has been working for awhile. we can switch back if it makes a difference, but doesn't seem too.